vision/spring-framework/src/main/resources/static/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Google Cloud Vision API with Spring code sample</title>
+</head>
+<body>
+
+<div>
+    <h1>Image Label Annotations</h1>
+    <p>Returns labels classifying the content of the image:</p>
+    <form action="/extractLabels">
+        Web URL of image to analyze:
+        <input type="text"
+               name="imageUrl"
+               value="https://upload.wikimedia.org/wikipedia/commons/3/35/Tux.svg"/>
```

Comment:
Are you intentionally using an `svg` image to demonstrate that vector graphics work as well?
It's probably not the typical use-case though.
BTW, it might be a good idea to use one of the [Google Doodles](https://www.google.com/doodles).

Reply:
Done.